### DISCUSSION TITLE
I don't understand if Vorta has a bug or if I'm configuring the program incorrectly

### DISCUSSION THREAD

**Discussion by fedegibut** [borgbase/vorta#2051] [FAQ]
**PROBLEM**
I would like the number of backups, after a week, to always be 7 (one done every night).
If I do a manual backup I expect to see 8 backups but after a week or a month, in the absence of manual backups, I expect to see 7 again. Always the 7 most recent backups obviously.
This is what I would like to do.

**TEST**
To test the functioning of Vorta I would prefer to think about the time. Example:
After 1 hour always 6 backups (one every 10 minutes).
If I do a manual backup I expect to see 7 backups but after an hour or after 2 hours, in the absence of manual backups, I expect to see 6 backups again. Always the 6 most recent backups obviously.
This is what I would like to do.

**CONFIGURATION**
I configured the program like this:

![Schermata del 2024-08-01 18-59-55](https://github.com/user-attachments/assets/bec40dcc-46b4-4482-a3b8-f5ed39ee73ce)
![Schermata del 2024-08-01 18-59-42](https://github.com/user-attachments/assets/46f542ac-f794-4830-ac34-f3c9bc23b0c5)
![Schermata del 2024-08-01 18-59-27](https://github.com/user-attachments/assets/5bf5ae69-5a95-40d0-9f4b-fc3d1fe3126f)
![Schermata del 2024-08-01 18-58-42](https://github.com/user-attachments/assets/53bc873d-9209-490b-93e8-14bdd74b2f61)
![Schermata del 2024-08-01 18-58-27](https://github.com/user-attachments/assets/d26bafe9-6e12-4de9-a3ba-c2ed180f1e6a)
![Schermata del 2024-08-01 18-58-08](https://github.com/user-attachments/assets/d929d94e-b051-465e-920a-c49522619d23)
![Schermata del 2024-08-01 18-57-56](https://github.com/user-attachments/assets/bd058e2f-9464-4c10-bd45-5b8e4abf39d6)
![Schermata del 2024-08-01 18-56-53](https://github.com/user-attachments/assets/7449a7aa-cfe2-4290-9724-8d73cd74be40)

_Thank you_

OS: **Ubuntu 24.04 LTS**

**Comment by jeop10** [comment]
Hello! 

First of all I am very new, and I am trying to learn how to use vorta and borg just like you, so take my answer just as a suggestion.

I believe that your problem is the "Archivi" > "Optioni pulizia ..."

In there you have to set the number of archives that you would like to keep, so for example, if you would like to see 7 daily (meaning 1 for each day) I think you need to set the number 7 in the daily textbox.

**Please, try this first in another repo, as I said I am just learning to use this tool**

Let me know if it works. 

Another thing that I have been reading is that one should run compact regularly to make sure that your repo does not hold empty references, which should lower the size of your repo. (I am still learning about this, so do your checking before doing it)

Regards.

**Comment by fedegibut** [comment]
Thanks for the text.
I can't wait a week to test Vorta.
Could you focus on the test setup?
If I put 6 in 'Hourly' the program saves 6 backups for each hour preceding the one in which I am performing the backup. If I set 0 Vorta should only save backups of the current time. Unfortunately this does not happen.

**Comment by jeop10** [comment]
I think with the current options of Vorta it's not possible to have "minutely" backups via the GUI

Maybe you can achieve that by running a cronjob every 10 minutes and the (maybe) doing a prune every hour to keep just the last backups of the last hour (I am not sure, please do your testing)

Another option could be using a different program to run your backups that allow minutely for example borgmattic https://torsion.org/borgmatic/docs/reference/configuration/

As I said before, please test this on a test repo. 

Good luck.

Regards

**Comment by fedegibut** [comment]
![Schermata del 2024-08-01 18-58-08](https://github.com/user-attachments/assets/ee2fc5b2-a865-4ff0-9b50-1196c1c2545d)
![Schermata del 2024-08-01 18-59-42](https://github.com/user-attachments/assets/d0f98878-eeba-40d5-b2ef-c68352aba745)
![Schermata del 2024-08-02 14-47-14](https://github.com/user-attachments/assets/a8a75d1b-c95a-4004-8e77-55f25df31aa8)

See how I configured the program's graphical interface. In my opinion it's a bug. There must be no backups before 2:00 pm. Too bad a Vorta expert isn't interested in my discussion.
